packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +254,26 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: any): any {
```

Comment:
The fields interface can be updated with `BraintreeHostedFieldsState['fields']`:

```suggestion
    private _mapErrors(fields: BraintreeHostedFieldsState['fields']): any {
```